src/Files.App/App.xaml.cs

```diff
@@ -197,7 +199,7 @@ private static async Task InitializeAppComponentsAsync()
 				);
 				await Task.WhenAll(
 					JumpList.InitializeAsync(),
-					ContextFlyoutItemHelper.CachedNewContextMenuEntries
+					addItemService.GetNewEntriesAsync()
```

Comment:
Might be better to do this the first time the user right clicks or opens the add item dialog, I don't think we need to do it on startup.

Reply:
I'm not sure why, but it doesn't work if we don't initialize it on startup (except `Ctrl+Shift+N`).
Right click and toolbar's `new item` won't work. Getting items should take less than 0.5s but actually Files freezes